presentation-compiler/src/main/dotty/tools/pc/completions/Completions.scala

```diff
@@ -544,14 +542,7 @@ class Completions(
         else false,
       )
       Some(search.searchMethods(query, buildTargetIdentifier, visitor).nn)
-    else
```

Comment:
Why was this removed? If I recon corretly, it was used when we couldn't correctly parse the file, so we've had  `Mode.None`?

Or is removing this a fix, so we don't get completions in comments?

Reply:
If we can't parse the file there are 2 options:
- parser crashed,
- parser recovered from an error, and we couldn't properly identify path.
The former is bigger problem, and this line wouldn't change a thing (it would still result in empty completions).
When latter happened, it is probably because of completely malformed code, like pasted from other language. I don't think this is an argument that will justify such completions. This line provided a lot of additional results in illegal context such as comments or:

```scala
def hello(a: String, u@@)
```
```scala
Option(1) match {
  case ma@@ =>
}
```

When the `CompletionMode.None` was actually derived correctly.
I don't think that this is the case we should think about, and even if someone pastes the code to modify / adjust it for scala, as soon as they start writing proper code completions pos should be derived correctly.

Reply:
Sounds good.